rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/domain/model/Transaction.java

```diff
@@ -790,15 +830,32 @@ public boolean afterLRANotification() {
 
                 while (listeners.hasNext()) {
                     URI uri = listeners.next();
+                    Response response = null;
+
+                    try {
+                        Future<Response> asyncResponse = client.target(uri)
+                                .request()
+                                .header(LRA.LRA_HTTP_ENDED_CONTEXT_HEADER, id)
+                                .async()
+                                .put(Entity.text(status.name()));
+
+                        response = asyncResponse.get(1000, TimeUnit.SECONDS);
```

Comment:
Can we also use PARTICIPANT_TIMEOUT here? I am not sure 1s is enough in normal network conditions.

Reply:
Modern networks are fast and a second is a long time.
LRA provides guarantees so if 1 s isn't enough it will be retried on the next recovery pass - ie a low value is transactionally safe.